src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar calendar--month-length_31 calendar--start-day_sun">
```

Comment:
```suggestion
    <div class="calendar calendar__month-length--31 calendar__start-day--sun">
```
it should be modificator